selfdrive/car/honda/carcontroller.py

```diff
@@ -134,9 +134,13 @@ def update(self, enabled, CS, frame, actuators, \
 
     # **** process the car messages ****
 
+    if CS.CP.carFingerprint in HONDA_BOSCH:
```

Comment:
No need to but behind if statement

Reply:
are you saying get rid of the `if`/`else` until we actually have more bosch longitudinal code? I was trying to make the changes in a way that merge conflicts are minimized (by inserting the conditional logic now)

Reply:
https://github.com/gregjhogan/openpilot/blob/f76ad2eac0cc63264e255b68944b849511f94ef1/selfdrive/car/honda/carcontroller.py#L142-L156

Reply:
I cleaned this up